### DISCUSSION TITLE
Celebrity Wall

### DISCUSSION THREAD

**Discussion by firiz** [WeAreMahsaAmini/Core#18] [Ideas]
A grid of Iran celebrities in black and white. They turn green or red based on their support. For instance, Ali Karimi would be green and Sepehr Khalse would be red. This will encourage others to stand with people and boots protestors morally.

**Comment by farnoodma** [comment]
Maybe "The Good, The Bad And The Ugly" instead of "green, red, and white" 😄

**Reply by nazunderground** [reply]
that's funny but I think its easier with color lol

**Reply by atriaa** [reply]
I agree. Differentiating with color makes it easier to scan and can show the progress that has been made. We can also create patterns with the colors as the images turn green (Ex. Mahsa's image or 'zan,zendegi,azadi').

**Reply by lvlast3l2** [reply]
keep it gray and green. this more meaning freedom and referendum.

if we want to make it right, we need to respect negative votes...

**Comment by Azadi2022** [comment]
great idea! also another wall for the most followed accounts on instagram, tiktok and twitter.

**Reply by atriaa** [reply]
+ Provide their account names or link out to their accounts, so people can easily click on their name and ask for support.

**Comment by nazunderground** [comment]
I think this is a great idea and I think we could also make something similar for magazines and news outlets. I've been aggressively tagging and leaving comments under the magazines that I like and follow. Some of them have already come out including; Vogue Arabia, Vogue America, W Magazine etc. thoughts?

**Comment by atriaa** [comment]
Once all celebrities on the wall have turned green we can do some fun visual treatment as a sign for one small celebration. We can then create a short clip with their videos, posts, notes, etc that supports this movement.

**Comment by haghi-paria** [comment]
How can I participate in this project?

I can be a help in UI/UX and front-end :)

**Comment by AnonymousMaximus2** [comment]
To Spread the word that world can hear you, 

you need money and register your organization to become recognized.

you need infrastructure for this idea,

and I explained this infrastructure here:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by SaeedGhaffari** [comment]
great idea. I'm developer, how i can help?

**Comment by Miladzio** [comment]
Hi all
A celebrity wall is an excellent idea. I love it but it's a little bit ambiguous. I'm gonna say, we should answer a couple of questions first. It will help us clarify the core idea behind it.  

1- What is its' core function? In the first stem, let's define the real 'need'. Showing who's supporting the movement and who is not? Are we looking for a map to distinguish celebrities?  Or do we want to create sth that documents a live 'history' of the movement based on famous people's support? Is it gonna be a persuasive product that leads new celebrities to take action? Is it a place for 'asking' passive people to become active? 

2- Who is our target audience? Yea, the answer is totally related to the 1st question! Politicians? Celebrities? International or local? Imagine if we target Goe Rogan and put his photo on the wall, how Amir Jafari's photo would be helpful on the page? 

Let's look at the idea as a startup. I know it's not gonna make money but it should have a real 'impact'. It's worth brainstorming about these points indeed.
If you agree, let's talk!

**Reply by ghost** [reply]
به نظرم سلبریتی های ایرانی، خیلی برای روحیه دادن به مردم داخل ایران مهمه، برای همین شاید بتونیم یه دیوار زیرمجموعه برای سلریتی های یارانی و به فارسی داشته باشیم که بشه در نهایت حتی یه صفحه اینستاگرام یا گروه تلگرامش کرد تا بشه بین مردم پخش بشه.

**Comment by nkouhi** [comment]
سلام، دمتون گرم
خیلی مهمه که یک وال هم برای سلبریتی های ایرانی باشه، حمایت اونها برای ما که داخل هستیم خیلی مهم تره، و هرکدام که حمایت میکنن کلی انرژی میاد،
اگر همین وال یک کپی هم داشته باشه برای ایرانی ها، همه وایرالش میکنیم.

**Comment by Setareh4IRAN** [comment]
Perfect, I can find these people and their other info. how can I help for making this list?

**Comment by fatihany98** [comment]
I have a suggestion, if it's possible to show each photo larger when you click on it, it would be great. otherwise, the photos are too small can difficult to recognize their face. in the end, this is just my idea, that could be any other thing instead of a pop-up window for better recognition. but I think the good point about this solution is that it can easily be done. 

thanks a lot

**Comment by lvlast3l2** [comment]
i think this is not good idea to paint celebrity's color in red. i think we need to respect people and their choice. just keep them gray. this is what iran goverment do by highlight disagree votes and who ever does.
if we want to make it right, we need act diffrent. not same as before. 
so please keep the colors on gray and green. this means respect to "NO" or any negative vote or diffrent ideas.... this means FREEDOM and what we want for our future and our kids and our IRAN

**Reply by lvlast3l2** [reply]
Persian:
موافق قرمز شدن سلبریتی ها نیستم. این همون واکنشی هستش که حکومت تو ایران به رای و نظر منفی داره و دقیقا مشکل ما هم با عدم شنیده شدن مخالفت هاست.  فقط کافیه رنگشون رو خاکستری نگه داریم. آزادی یعنی احترام گذاشتن به نظر های متفاوت.
اگر میخواهیم کار درست رو انجام بدیم، نباید مثل قبل به دنبال حذف و اجبار باشیم. باید جایگاهی هم برای مخالفت قائل باشیم و فکر میکنم خاکستری و سبز موندن سلبریتی ها، تجربه قشنگتری رو برای آیندگانمون به جا میذاره و بهتر معنای آزادی داشتن رو بیان میکنه. و این چیزیه که برای ایران ، برای بچه هامون ، برای خانواده هامون در تلاشیم.

**Reply by Homap** [reply]
I completely agree! We want to create a country where opinions are free and respected and this is our start.

**Reply by jessgess** [reply]
I fully agree and I would go one step further by removing the red/gray section all together! The inclusion argument is correct but iI also think it is not feasibly possible to include pictures of all celebrities in the world who might have not engaged or supported the movement yet. So inevitably a bunch will be left out. And if they are missed out they might not feel an urge to support. Plus there is no need to put their names and pressure them to support if its not something that resonates with them. 

I also heard another idea about adding the functionality that clicking on the gray/red celeb pics would create a tweet addressing them and requesting support. This I think is a super bad idea. That will definitely backlash as spamming anyone into doing anything will not work and will in fact have the reverse effect.  They might even go public and say stop spamming me or something like that!

**Reply by Jenesyx** [reply]
I agree

**Comment by fatihany98** [comment]
you can add Jessica Chastain in Wall with green color:). she is an American actress and producer that support us. 
thanks

**Comment by Kour9sh** [comment]
**The Colour Code Idea For The Designers**  (Our Flag Colours)

Green : **#239F40**
White : **#FFFFFF**
Red    : **#DA0000**

**Reply by Kour9sh** [reply]
🟢 **GREEN**  :  **Hex : 239F40**  |  **RGB : 35, 159, 64**  |  **CMYK : 100, 0, 67, 32**,  |  **Pantone : 335**  |  **RAL : 6032**

⚪️ **WHITE** : **Hex : FFFFFF**  |  **RGB : 255, 255, 255**  |  **CMYK : 0, 0, 0, 0,**  |  **Pantone : N.A.**  |  **RAL : N.A.** 

🔴 **RED** : **Hex : DA0000**  |  **RGB : 218, 0, 0,**  |  **CMYK : 0, 100, 80, 5**  |  **Pantone : 186**  |  **RAL : 3020**

**Comment by nikign** [comment]
Can we consider also adding a print view of green celebrities and a simple list view of activities? It will be helpful if people want to share offline for encouragement.

**Comment by ghost** [comment]
من یه لیست دارم از اسامی و لینکشون و اینا، داشتم جمع میکردم، اگر به درد میخوره میتونم براتون بفرستم.

**Comment by mkianpour** [comment]
Is it based on a voting system or something? I mean we can easily say Ali Karimi is green but for example, IMO: what Karim Ansarifard said yesterday is dictated by government and makes him red. Who will decide?

**Comment by daneial** [comment]
If you need a product designer, I'm here to help.

**Comment by parisamoshfegh** [comment]
frontend if needed!

**Reply by farnoodma** [reply]
@paria340 please join us [here](https://discord.gg/RFggewhk).

**Comment by h-agharezaei** [comment]
I'm here to help with the backend

**Reply by farnoodma** [reply]
Hey, please join us [here](https://discord.gg/6pqqqZgGnq).

**Reply by h-agharezaei** [reply]
joined!
thanks

**Comment by Jenesyx** [comment]
Hello everyone!
I'm the creator of the Woman Life Freedom logo, I can also help in UI UX and Front-end and some Graphic design :))

**Reply by farnoodma** [reply]
Hey, please join us [here](https://discord.gg/6pqqqZgGnq).

**Comment by ahmadezoji** [comment]
How can i help?? Im mobile application developer . I can make react native application for this.

**Reply by farnoodma** [reply]
Hey, please join us [here](https://discord.gg/6pqqqZgGnq).

**Reply by Jenesyx** [reply]
already did
